app/models/form_profiles/va526ez.rb

```diff
@@ -93,7 +93,10 @@ def initialize_vets360_contact_info(user)
     {
       mailing_address: convert_vets360_address(vet360_contact_info.mailing_address),
       email_address: vet360_contact_info.email.try(:email_address),
-      primary_phone: [vet360_contact_info.home_phone.area_code, vet360_contact_info.home_phone.phone_number].join('')
+      primary_phone: [
+        vet360_contact_info.home_phone.try(:area_code),
+        vet360_contact_info.home_phone.try(:phone_number)
```

Comment:
The `try` operation works just fine but usually I use the safe navigation operator `&.` when trying to access data like this. The difference, as I understand it, is that `try` will ignore basically all exceptions but `&.` will raise a `NoMethodError` if the method isn't `nil` or hasn't been implemented.